openapi-cli/src/test/resources/generators/client/ballerina/deprecated_functions.bal

```diff
@@ -0,0 +1,91 @@
+import ballerina/http;
+import ballerina/url;
+
+public isolated client class Client {
+    final http:Client clientEp;
+    # Gets invoked to initialize the `connector`.
+    #
+    # + clientConfig - The configurations to be used when initializing the `connector`
+    # + serviceUrl - URL of the target service
+    # + return - An error if connector initialization failed
+    public isolated function init(http:ClientConfiguration clientConfig =  {}, string serviceUrl = "https://petstore.swagger.io:443/v2") returns error? {
+        http:Client httpEp = check new (serviceUrl, clientConfig);
+        self.clientEp = httpEp;
+    }
+    # List all pets
+    #
+    # + 'limit - How many items to return at one time (max 100)
+    # + return - An paged array of pets
+    #
+    # # Deprecated
```

Comment:
why does it have double "#" there, is there any reason for it?

Reply:
It is how Ballerina accepts comments related to `Deprecated` annotations. Removing one `hash` token will be resulted in syntax errors. Please see ballerina by example [here](https://ballerina.io/1.2/learn/by-example/deprecation).